salt/orch/update.sls

```diff
@@ -125,10 +126,11 @@ update_modules:
     - tgt: {{ worker_id }}
     - sls:
       - kubelet.stop
-      - kube-proxy.stop      
+      - kube-proxy.stop
       - docker.stop
       - flannel.stop
       - etcd.stop
+      - container-feeder.stop
```

Comment:
I would stop `container-feeder` before `docker`, just in case...

Reply:
This is on an already running machine, so container-feeder should not be running, but I will update the PR, makes sense.